### PR TITLE
add graduation report field

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by johngruen** [comment]
Fixes #1753

**Comment by johngruen** [comment]
Please note, there's no l10n here yet...This is to accommodate the Universal Search case and we should discuss the best l10n method for these reports in #1829

File: docs/content/reference.md [hunk 1]
```diff
@@ -233,10 +233,10 @@ modified: '2016-01-01T00:00:00.000000Z'
 
 ## `completed`
 
-UTM-formatted date indicating the time the experiment will be retired. Used to calculate labels like "Ending Soon". Optional.
+UTM-formatted date indicating the time the experiment will be retired. Used to calculate labels like "Ending Soon". Optional. Note: should include timezone to ensure experiment completes at the same time everywhere.
 
 ```yaml
-completed: '2016-01-01T00:00:00.000000Z'
+completed: '2016-01-01T00:00:00.000000Z+00:00'
```
Comment (dannycoates) [inline docs/content/reference.md]: fyi 'Z' **is** the UTC timezone, so adding `+00:00` after it makes it invalid.

**Review by dannycoates — CHANGES_REQUESTED** [review]
Looks good aside from the `completed` example

File: docs/content/reference.md (hunk 1 above)
Comment (johngruen) [reply docs/content/reference.md]: I really need to read the docs about time related stuff

**Comment by johngruen** [comment]
@dannycoates that should do it

**Review by dannycoates — APPROVED** [review]

**Pull request merged**